main.py

```diff
@@ -10,4 +10,5 @@ def b():
     print( " " + str(a))
 
 b()
-print(a)
```

Comment:
Try to remove unintended code changes from your pull request. Just a good habit. (change it back to how it was before)